server.js

```
@@ -0,0 +1,221 @@
+const http = require('http');
+const fs = require('fs');
+const PORT = 8080;
+const querystring = require('querystring');
+let date = new Date().toUTCString();
+const admin = {
+    bri: 'password',
+}
+
+const server = http.createServer((req, res) => {
+    // console.log(req);
+    // console.log('reqmethod', req.method);
+    // console.log('requrl', req.url);
+    let body = '';
+    req.on('data', (chunk) => {
+        body += chunk;
+    });
+    req.on('error', (error) => {
+        console.error(error);
+    })
+    req.on('end', () => {
+        let urlType;
+        let URL = req.url;
+        // GET METHOD
+        if (req.method === 'GET') {
+            if (req.url === '/' || req.url === '') {
+                urlType = 'html';
+                URL = '/index.html';
+            } else {
+                urlType = req.url.split('.');
+                urlType = urlType[1];
+            }
+            fs.readdir(`./public/`, (err, files) => {
+                if (files.includes(URL.slice(1)) === false) {
+                    URL = `/404.html`;
+                    urlType = 'html';
+                }
+                fs.readFile(`./public${URL}`, (err, data) => {
+                    if (err) {
+                        return console.log('Could not write the file.');
```

This should return an error to the client using either a 400/500 error